superset/security/manager.py

```diff
@@ -604,6 +600,19 @@ def user_view_menu_names(self, permission_name: str) -> Set[str]:
             return {s.name for s in view_menu_names}
         return set()
 
+    def get_accessible_databases(self) -> List[int]:
+        """
+        Return the list of databases accessible by the user.
+
+        :return: The list of accessible Databases
+        """
+        perms = self.user_view_menu_names("database_access")
+        return [
+            int(match[1])
+            for perm in perms
+            if (match := DATABASE_PERM_REGEX.match(perm))
```

Comment:
You could assign the group a name via the `?P` which might improve the readability here.

Reply:
TIL! 💡